lib/rules/rules.json

```diff
@@ -198,7 +198,7 @@
     },
     "uniq": {
         "compatible": true,
-        "alternative": "[... new Set(arr)]",
+        "alternative": "Array.prototype.filter((v, i, a) => a.indexOf(v) === i)",
```

Comment:
Why? this guide is not only for IE. Supporting IE is nice to have

Reply:
still hasn't actually been changed after review requested